portfolio/src/main/java/com/google/sps/servlets/TranslateAPIServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import java.io.IOException;
+import java.util.HashSet;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslateAPIServlet extends HttpServlet{
+
+    HashSet<String> acceptedLanguages = new HashSet<String>();
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        acceptedLanguages.add("en");
+        acceptedLanguages.add("es");
+        acceptedLanguages.add("fr");
```

Comment:
I'll accept this for now. But would be good to put this in the constructor instead of in the doPost.

Reply:
Ohhh thank you! I was wondering how I could have it be instantiated when the class is used. I'll make a commit for this after merging!